spec/models/miq_request_workflow_spec.rb

```diff
@@ -399,4 +399,23 @@
       expect(workflow.cast_value(1, :other)).to      eq(1)
     end
   end
+
+  context "#storage_to_hash_struct" do
+    before do
```

Comment:
can be better written as
```ruby
let (:storage) { FactoryGirl.create(:storage_vmware) }
```

I like [betterpecs.org](http://betterspecs.org/#let), its a great resource